src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -127,6 +128,18 @@ class LinalgBackendEigen : public LinalgBackendBase
 		return c;
 	}
 
+	/** Eigen3 matrix C = alpha*A + beta*B method */
+	template <typename T>
+	SGMatrix<T> add_impl(const SGMatrix<T>& a, const SGMatrix<T>& b, T alpha, T beta) const
```

Comment:
@OXPHOS Please add to your TODO list (in addition to deplying these updates) that we want to document things also in a readme, in particular the inplace example by @vigsterkr above should be in there to make the point that we can do inplace, but it will be slower, so we only wanna do it if memory blows up otherwise (e.g. element wise operations on gigantic matrices or so)....